websms.py

```diff
@@ -0,0 +1,443 @@
+import json
+import logging
+from typing import List, Optional, Tuple, Union
+
+import requests
+
+logger = logging.getLogger('websms')
+
+
+class SMSServiceError(Exception):
+    """Base exception class."""
+
+    def __init__(
+        self,
+        message: Optional[str] = None,
+        original_exception: Optional[
```

Comment:
IMO it can be replaced by `Optional[requests.RequestException]`, because `requests.HTTPError` is its subclass.

Reply:
sounds good! Thanks for the tip.